protocol/contracts/leaser/src/msg.rs

```diff
@@ -84,7 +84,14 @@ pub enum SudoMsg {
         lease_due_period: Duration,
     },
     CloseProtocol {
+        // Since this is an external system API we should not use [Code].
+        new_lease_code_id: Uint64,
         migration_spec: ProtocolContracts<MigrationSpec>,
+        /// If `ForceClose::KillProtocol` closes the protocol even if it has not closed leases
```

Comment:
`If`? I'm guessing this happened due to rewording it later.

Reply:
nice catch, thanks!